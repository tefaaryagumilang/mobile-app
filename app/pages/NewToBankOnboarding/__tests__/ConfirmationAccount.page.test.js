import React from 'react';
import renderer from 'react-test-renderer';
import ConfirmationAccount from '../ConfirmationAccount.page';
import {Provider} from 'react-redux';
import {createStore} from 'redux';
import {reduxForm} from 'redux-form';

const store = createStore(() => ({}));
const DecoratedForm = reduxForm({form: 'testForm'})(ConfirmationAccount);

describe('NTB OnboardingConfirmationAccount page', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Provider store={store}><DecoratedForm/></Provider>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
