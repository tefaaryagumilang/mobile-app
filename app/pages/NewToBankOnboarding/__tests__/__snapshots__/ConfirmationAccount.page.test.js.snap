// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NTB OnboardingConfirmationAccount page renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "flex": 1,
      "justifyContent": "space-between",
      "paddingHorizontal": 20,
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "flex-end",
        }
      }
    >
      <Image
        source="banksinarmas.png"
      />
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": undefined,
        "backgroundColor": "transparent",
        "flex": 1,
        "justifyContent": "center",
        "paddingHorizontal": 30,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#000000",
          "fontSize": 24,
          "paddingBottom": 100,
        }
      }
    >
      Apakah Anda memiliki akun bank Sinarmas?
    </Text>
  </View>
  <View
    style={
      Object {
        "paddingBottom": 20,
      }
    }
  >
    <View
      style={
        Object {
          "paddingBottom": 20,
        }
      }
    >
      <View
        accessible={true}
        clickable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#FA3F70",
                "borderRadius": 30,
                "flexDirection": "row",
                "height": 47,
                "justifyContent": "center",
                "paddingHorizontal": 15,
              },
              Object {
                "borderColor": "#ED1D25",
                "borderRadius": 50,
                "borderWidth": 2,
              },
              Object {},
            ]
          }
        >
          <Text
            style={
              Object {
                "color": "#FFFFFF",
                "fontSize": 16,
              }
            }
          >
            YA, SAYA PUNYA
          </Text>
        </View>
      </View>
    </View>
    <View
      accessible={true}
      clickable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "#FA3F70",
              "borderRadius": 30,
              "flexDirection": "row",
              "height": 47,
              "justifyContent": "center",
              "paddingHorizontal": 15,
            },
            Object {
              "borderColor": "#ED1D25",
              "borderRadius": 50,
              "borderWidth": 2,
            },
            Object {},
          ]
        }
      >
        <Text
          style={
            Object {
              "color": "#FFFFFF",
              "fontSize": 16,
            }
          }
        >
          TIDAK, BUAT AKUN BANK
        </Text>
      </View>
    </View>
  </View>
</View>
`;
